offline data made and updated other functions accordingly

philharr opened the pull request.

Commit:
offline data made and updated other functions accordingly

philharr committed:
Delete .DS_Store

philharr committed:
Delete conformal_interestingness_model-checkpoint.py

philharr committed:
Delete conformal_model-checkpoint.py

philharr committed:
Delete conformal_model_data-checkpoint.py

philharr committed:
Delete conformal_model_offline_data-checkpoint.py

philharr committed:
Delete time_series_features-checkpoint.py

philharr committed:
Delete .DS_Store

benblamey commented on haste/windowed_conformal_model/conformal_interestingness_model.py:
```diff
@@ -139,8 +141,8 @@ def interestingness(self,
-
-            # p_values = conformal_interestingness(ALL_FEATURES, ts_features, ALL_Y)
+            ts_features_standardized = __standardize(ts_features, STANDARDIZING_VALUES[:, :, timestamp - 1])
+            # p_values = conformal_interestingness(TRAIN_FEATURES_STANDARDIZED[:, :, timestamp - 1], ts_features_standardized, TRAIN_Y)
```
can we enable the model here?

philharr replied:
Yep, give it a shot!

benblamey commented on haste/windowed_conformal_model/time_series_features.py:
```diff
@@ -28,15 +26,12 @@ def time_series_features(feature_time_series, timestamps, end_time):
-    feat[1] = np.std(Y)  # standard deviation of raw data
-
-    y_pred = gam.predict(pd.DataFrame(np.arange(end_time)))
+    z = np.polyfit(timestamps, feature_time_series, min(8, round(len(timestamps)/3)))
```
is the 8 here the window size? can we reference the constant?

philharr replied:
No, it's a separate thing. So it stays as 8 in all cases for the time being.

Hakanwie replied:
min(8, round(len(timestamps)/3) is the degree of the polynomial we want to fit so its kind of a "hack" until we have a better solution, but it works for now

benblamey commented:
also suggest rebasing and squashing the commits - but don't worry about it if you don't know how

philharr commented:
Not sure how to rebase and squash, so will leave that for you. Perhaps you can teach me how to do it later?

benblamey commented on haste/windowed_conformal_model/conformal_interestingness_model.py:
```diff
@@ -139,8 +141,8 @@ def interestingness(self,
 
-
-            # p_values = conformal_interestingness(ALL_FEATURES, ts_features, ALL_Y)
+            ts_features_standardized = __standardize(ts_features, STANDARDIZING_VALUES[:, :, timestamp - 1])
```
hmmm.. this will fail if the timestamp is too high (for another dataset). maybe we should check for that - and fallback to interestingness 1?

philharr replied:
wait a sec, that should be 
STANDARDIZING_VALUES[:, :, (timestamp / WINDOW_SIZE) - 1]
and similarly in the following part for the p_values change to
TRAIN_FEATURES_STANDARDIZED[:, :, (timestamp / WINDOW_SIZE) - 1]
sorry about that

philharr committed:
Update conformal_interestingness_model.py

fixed indexing in two places (scaling by WINDOW_SIZE) and activated conformal_interestingness!
